packages/addon-dev/src/rollup.ts

```diff
@@ -57,8 +57,8 @@ export class Addon {
     return hbs();
   }
 
-  gjs() {
-    return gjs();
+  gjs(options?: { inline_source_map: true }) {
+    return gjs(options);
```

Comment:
Is there any downside to defaulting this to true?

We already have rollup's sourcemaps enabled

Reply:
hmm well probably not, I think I can default it to true in this case if you want @NullVoxPopuli

Reply:
I changed the default to true @NullVoxPopuli

Reply:
I mean, in your defense, we don't have sufficient testing if for addon-dev in this repo